docs/source/installation.rst

```diff
@@ -100,6 +100,16 @@ If you prefer a ``pip`` installation, we provide the following command
 Note that, differently from the  ``conda`` command, the above **will not** create a virtual environment.
 Make sure you create and activate your environment previously.
 
+Enable Nvidia Collective Communication Library (NCCL)
```

Comment:
I would move this above, before the `Step-by-step installation for users` section and say that to enable this two additional dependencies are required, cupy and nccl (basically to tie this with the first part of this page that talks about dependencies...